services/src/main/java/org/keycloak/services/resources/account/AccountRestService.java

```diff
@@ -405,26 +405,20 @@ public LinkedAccountsResource linkedAccounts() {
         checkAccountApiEnabled();
```

Comment:
Yeah, I like this approach

Reply:
EDIT: this breaks several tests in AccountRestServiceTest because the intermediary inUseClients, offlineClients and consentModels are not fully formed when the last map operation is performed. As the stream elements are processed one by one these intermediary collections that are needed to convert to ClientRepresentation are not totally in place by the time the modelToRepresentation call is made.

Because of that I had to collect each of the intermediary streams in order to force the processing of the intermediary collections and ensure they are fully formed when modelToRepresentation is called on each client.